google/cloud/options.h

```diff
@@ -209,6 +210,7 @@ class Options {
   friend Options internal::MergeOptions(Options, Options);
   friend void internal::CheckExpectedOptionsImpl(
       std::set<std::type_index> const&, Options const&, char const*);
+  friend bool internal::IsEmpty(Options const&);
```

Comment:
Did you consider just an `Options::empty()`?

Reply:
I don't want to grow the public API. Eventually, I want to remove this function. I should probably attach an issue to it, to help me remember my intentions...